### PR TITLE
fix: corrected typo in HTML Forms challenge Step 8

### PR DESCRIPTION
**Pull request by pramod74**
Checklist:

- [x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x ] My pull request targets the `main` branch of freeCodeCamp.
- [x ] I have tested these changes either locally on my machine, or GitPod.

I have replaced the word 'relative' with the word 'equal' in the first statement of Step 8 of 'Learn HTML Forms by Building a Registration Form' challenge.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ dashedName: step-8
 ---
 
 # --description--
-The `vh` unit stands for viewport height, and is relative to 1% of the `height` of the viewport.
+The `vh` unit stands for viewport height, and is equal to 1% of the `height` of the viewport.
```
[the remark this answers is not in the document]
Comment (jdwilkin4) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md:9]: I think both versions are technically correct. So I wouldn't consider this a typo. 
But from a learner perspective, I can see how using the word equal would add more clarity to the description of the `vh` unit and create a better understanding of the concept.

[the remark this answers is not in the document]
Comment (JoyShaheb) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md:9]: i would approve this @jdwilkin4

**Review by JoyShaheb — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdwilkin4) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md:9]: Sure, we can meet in the middle and mention both

cc @pramod74 Can you make this change?

[the remark this answers is not in the document]
Comment (jdwilkin4) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md:9]: works for me

[the remark this answers is not in the document]
Comment (pramod74) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f1922fcbd2410527b3bd89.md]: Thanks. I have updated my PR. Request you to review the same.

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**